test/specs/scenarios/document.js

```diff
@@ -141,5 +142,27 @@ export const documentScenarios = async (lang) => {
       uploadFileAndClickConfirmButton(documentUpload, confirm, 'face.jpeg')
       verificationComplete.verifyUIElements(copy)
     })
+
+    // it('should verify accessibility for the uploader screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+    //   runAccessibilityTest(driver)
+    // })
+
+    // it('should verify accessibility for the document upload confirmation screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+  
+    //   documentUpload.getUploadInput()
+    //   documentUpload.upload('passport.jpg')
+    //   runAccessibilityTest(driver)
+    // })
+
+    it('should verify accessibility for verification complete screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}&oneDoc=true&async=false&useWebcam=false`)
```

Comment:
Minor: And again here. Can be written as `${localhostUrl}?language=${lang}&oneDoc=true&async=false&useWebcam=false`